strawberry/extensions/context.py

```diff
@@ -56,10 +58,13 @@ def __init_subclass__(cls) -> None:
     LEGACY_ENTER: str
     LEGACY_EXIT: str
 
-    def __init__(self, extensions: List[SchemaExtension]) -> None:
+    def __init__(
+        self, extensions: List[SchemaExtension], execution_ctx: ExecutionContext
```

Comment:
```suggestion
        self, extensions: List[SchemaExtension], execution_context: ExecutionContext
```

let's keep the long name, since we use that in other places too 😊